src/main/kotlin/org/blueliner/springbootwithkotlin/controller/JokeController.kt

```diff
@@ -0,0 +1,32 @@
+package org.blueliner.springbootwithkotlin.controller
+
+import org.blueliner.springbootwithkotlin.dto.JokeDto
+import org.blueliner.springbootwithkotlin.service.JokeService
+import org.springframework.ai.chat.model.ChatResponse
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.Optional
+
+
+@RestController
+@RequestMapping("/jokes")
+class JokeController(private val jokeService: JokeService) {
+
+    @GetMapping("/ai-format")
+    fun getJokeUsingBasicAiModelOutput(): ResponseEntity<ChatResponse> {
+        return ResponseEntity.of(Optional.of(jokeService.getJokeUsingBasicAiModelOutput()));
```

Comment:
one of the most important features kotlin has is simplified null safety: https://kotlinlang.org/docs/null-safety.html
you don't ever need to use optional in kotlin code, "ChatResponse" means it's non-null and "ChatResponse?" means its nullable